Cargo.toml

```
@@ -21,3 +21,5 @@ codegen-units = 1
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
+orchard = { git = "https://github.com/daira/orchard.git", rev = "72e4cce7aa5f18bfb50249c0ccad10b848a8077a" }
+group = { git = "https://github.com/zkcrypto/group.git", rev = "bbd331f564a05d516d7dd81a1ceb0e431d264546" }
```

```suggestion
group = { git = "https://github.com/zkcrypto/group.git", rev = "a7f3ceb2373e9fe536996f7b4d55c797f3e667f0" }
```
(This is the merge commit of https://github.com/zkcrypto/group/pull/36 . bbd331f was an intermediate commit, even though it did include the necessary changes. )